### PR TITLE
Update entry-machines.md

### PR DESCRIPTION
**Pull request by PWEB18**
updated bambino entries given latest understanding
still unsure about PI times with both models, and why the Plus comes with a cleaning disc but not the standard

### CODE REVIEW

File: recommendations/entry-machines.md [hunk 1]
```diff
@@ -73,20 +73,23 @@ Breville machines come packed with features out of the box at a range of prices.
...
-- Pre-infusion is not as flexible as the other Breville machines listed.
+- Pre-infusion is not as flexible as Breville thermoblock machines (limited to 10 seconds)
 
 ### **Breville Bambino Plus** ([$500](https://www.breville.com/us/en/products/espresso/bes500.html))
+### This is the same as  Bambino with slightly better build quality and the addition of an automatic milk steaming wand with adjustable temperature and texture.
 #### Pros:
-- PID Controlled temp (single setting, 200F)
-- Has a robust automatic steaming wand, making steaming a foolproof process
-- Manual pre-infusion possible, unlike the basic Bambino
+- Same as Bambino including:
```
Comment (skatcat31) [inline recommendations/entry-machines.md:85]: should this have a the in it?
```suggestion
- Same as the Bambino including:
```

File: recommendations/entry-machines.md [hunk 2]
```diff
@@ -73,20 +73,23 @@ Breville machines come packed with features out of the box at a range of prices.
...
+- A robust automatic steaming wand, making steaming a foolproof process
```
Comment (skatcat31) [inline recommendations/entry-machines.md:86]: Grammarly suggestion

```suggestion
- A robust automatic steaming wand that makes steaming a foolproof process
```

File: recommendations/entry-machines.md [hunk 3]
```diff
@@ -73,20 +73,23 @@ Breville machines come packed with features out of the box at a range of prices.
...
+- 64oz water jug compared to 47 oz of the Bambino
+- Comes with cleaning tool/disc for easier maintenance
 
 #### Cons:
-- Higher cost than the regular bambino.
+- Higher cost than the regular bambino
+- Pre-infusion is not as flexible as Breville thermoblock machines (limited to 10 seconds)
```
Comment (skatcat31) [inline recommendations/entry-machines.md:92]: isn't this the same as the regular bambino? If so why not mention it that way?

```suggestion
- Pre-infusion is the same as the regular Bambino (limited to 10 seconds)
```

**Review by skatcat31 — COMMENTED** [review]
Not a bad change. Grammarly had some suggestions but as you can tell they're pretty minor

File: recommendations/entry-machines.md (hunk 3 above)
Comment (PWEB18) [reply recommendations/entry-machines.md:92]: the two bambinos are the same (and the same to other thermojets); this is a con compared to an infuser or BBE purchase

Comment (skatcat31) [reply recommendations/entry-machines.md:92]: correct, but does it need to be noted twice if it's the same as above for the regular Bambino?

Comment (PWEB18) [reply recommendations/entry-machines.md:92]: oh I see what you mean, got it

**Pull request merged**